src/test/resources/grader/fixtures/Config/ExternalProcessEndSignalConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package delft;
+
+import nl.tudelft.cse1110.andy.config.RunConfiguration;
+import nl.tudelft.cse1110.andy.config.MetaTest;
+import nl.tudelft.cse1110.andy.execution.process.ExternalProcess;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Configuration extends RunConfiguration {
+
+    @Override
+    public List<String> classesUnderTest() {
+        return List.of("delft.EmptyLibrary");
+    }
+
+    @Override
+    public Map<String, Float> weights() {
+        return new HashMap<>() {{
+            put("coverage", 0.0f);
+            put("mutation", 0.0f);
+            put("meta", 1.0f);
+            put("codechecks", 0.0f);
+        }};
+    }
+
+    @Override
+    public ExternalProcess externalProcess() {
+        String tmp = System.getProperty("java.io.tmpdir");
+        return new ExternalProcess(
+                "sh " + tmp + "/andy_test_external_process_end_signal.sh",
```

Comment:
Oh that's a good thing to consider. The script will also pass the directory in which the sh is, ok, that should work!